### PR TITLE
added models and created migrations

### PR DESCRIPTION
**Pull request by taraskonchak**
Added models for:
     - Users,
     - Roles,
     - Account,
     - Transactions
Created migrations for this models.
Please run "pip install -r requirements.txt" before migrate. Also please check your Also please check your postgres password and change it.

### CODE REVIEW

File: MiniBank/settings.py [hunk 1]
```diff
@@ -75,8 +77,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'MiniBank',
+        'USER': 'postgres',
+        'PASSWORD': 'newPassword',
```
Comment (ement06) [inline MiniBank/settings.py:83]: could we use environment variables instead of hardcoded?

File: requirements/requirements.txt [hunk 2]
```diff
@@ -2,3 +2,4 @@ asgiref==3.2.5            # via django
 django==3.0.4             # via -r requirements/requirements.in
 pytz==2019.3              # via django
 sqlparse==0.3.1           # via django
+psycopg2==2.7.7
```
Comment (ement06) [inline requirements/requirements.txt:5]: move psycopg2 to requirements/requirements.in and run pip-compile

Comment (ement06) [reply requirements/requirements.txt:5]: also, add to requirements/test_requirements.in

**Comment by ement06** [comment]
Please, write unit tests for checking connections to the database.

**Review by ement06 — CHANGES_REQUESTED** [review]

File: accounts/models.py [hunk 3]
```diff
@@ -1,3 +1,28 @@
...
+    credit_percentage = models.FloatField()
+    deposit_percentage = models.FloatField()
+    credit_start_date = models.DateTimeField()
+    credit_end_date = models.DateTimeField()
+    deposit_start_date = models.DateTimeField()
+    deposit_end_date = models.DateTimeField()
+    user = models.ForeignKey(Users, on_delete=models.CASCADE)
+
+
+class Transactions(models.Model):
```
Comment (olyapetryshyn) [inline accounts/models.py:24]: should be in single form (not plural)

File: accounts/models.py [hunk 4]
```diff
@@ -1,3 +1,28 @@
 from django.db import models
 
-# Create your models here.
+from users.models import Users
+
+
+class Account(models.Model):
```
Comment (olyapetryshyn) [inline accounts/models.py:6]: you should inherit from TimeStampedModel (each model)

File: users/models.py [hunk 5]
```diff
@@ -1,3 +1,19 @@
 from django.db import models
 
-# Create your models here.
+
+class Users(models.Model):
```
Comment (olyapetryshyn) [inline users/models.py:4]: same here (single form) and inheritance from AbstractBaseUser etc.

File: users/models.py [hunk 6]
```diff
@@ -1,3 +1,19 @@
...
+    first_name = models.CharField(max_length=30)
+    last_name = models.CharField(max_length=30)
+    email = models.EmailField(max_length=70, unique=True)
+    phone_number = models.CharField(max_length=13, unique=True)
+    passport_number = models.CharField(max_length=8, unique=True)
+    identity_code = models.CharField(max_length=15, unique=True)
+    company = models.ForeignKey("self", on_delete=models.CASCADE)
```
Comment (olyapetryshyn) [inline users/models.py:11]: does this mean that when we delete a company - we delete all users? we discussed that it should not be like that. Instead, use on_delete=DO_NOTHING

File: accounts/models.py [hunk 7]
```diff
@@ -1,3 +1,28 @@
...
+    number = models.CharField(max_length=16, unique=True)
+    is_debit = models.BooleanField()
```
Comment (olyapetryshyn) [inline accounts/models.py:8]: add default=True

File: accounts/models.py [hunk 8]
```diff
@@ -1,3 +1,28 @@
...
+    CVV = models.IntegerField()
+    pincode = models.IntegerField()
+    expiration_date = models.DateField()
+    amount = models.FloatField()
+    credit_limit = models.FloatField()
+    deposit_amount = models.FloatField()
```
Comment (olyapetryshyn) [inline accounts/models.py:14]: add constraints to deposit and credit fields. It may be null too.

File: MiniBank/settings.py [hunk 9]
```diff
@@ -31,6 +31,8 @@
 # Application definition
 
 INSTALLED_APPS = [
+    "accounts",
+    "users",
```
Comment (olyapetryshyn) [inline MiniBank/settings.py:35]: please, add common app here too;)

**Pull request closed without merging**